10b-heritability_aar.sh

```diff
@@ -18,7 +18,7 @@ for clock_name in $clock_names
 do
   echo "Runing SNP heritability for " $clock_name $i
   ${gcta} \
-          --grm ${grmfile_all}  \
+          --grm ${grmfile_all}_gaws10  \
```

Comment:
should this be _gwas10?

Reply:
Yes, ${grmfile_all}_gaws10 filters out the related individuals based on the threshold 0.05, while ${grmfile_all} didn't.